lib/actions/appleMusic.js

```diff
@@ -155,15 +145,16 @@ function appleMusic(player, values) {
                 return player.coordinator.clearQueue()
                   .then(() => player.coordinator.addURIToQueue(queueTracks[0].uri, queueTracks[0].metadata, true, 1))
                   .then(() => {
-                      if (radioOn(player)) {
-                        return promise.then(() => player.coordinator.setAVTransport(queueURI,''));
+                      if (isRadioOrLineIn(player.coordinator.state.currentTrack.uri)) {
```

Comment:
I think so, I believe currentTrack.uri could point to an actual http streaming url which would be impossible to predict.

If you are only trying to avoid setting queue again, you could just check if it startsWith `x-rincon-queue:`